### ISSUE TITLE
Remove deprecated twisted.application.strports.parse

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8926]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8926|
|Type|enhancement|
|Created|2016-11-27 00:10:40Z|
|Branch|https://github.com/twisted/twisted/tree/8926-rem-strportparse|

<details><summary>Searchable metadata</summary>

```
trac-id__8926 8926
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__8926_rem_strportparse 8926-rem-strportparse
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1480205440018672 1480205440018672
changetime__1480404726517730 1480404726517730
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

PR Available at https://github.com/twisted/twisted/pull/610

**Comment by twisted-trac** [comment]
|[<img alt="moshez's avatar" src="https://avatars.githubusercontent.com/u/422703?s=50" width="50" height="50">](https://github.com/moshez)|@moshez set owner to @hawkowl|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset 160ecab6505e6634a8ea5c33fa209f90ffd5e52d
```
#!CommitTicketReference repository="" revision="160ecab6505e6634a8ea5c33fa209f90ffd5e52d"
Merge 8926-rem-strportparse: Remove twisted.application.strports.parse

Author: hawkowl
Reviewer: moshez
Fixes: #8926
```
